<ui version="4.0" >
 <class>InformationPanel</class>
 <widget class="QWidget" name="InformationPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Information Panel</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="iconLabel" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap>../../../../../../home/kde-devel/ark-trunk/pics/ox32-action-ark_view.png</pixmap>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fileName" >
     <property name="text" >
      <string>&lt;font size=&quot;+2&quot;>&lt;b>Filename&lt;/b>&lt;/font></string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="additionalInfo" >
     <property name="text" >
      <string>Unknown file type</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="firstSeparator" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="metadataLabel" >
     <property name="text" >
      <string>Metadata Label</string>
     </property>
     <property name="margin" >
      <number>10</number>
     </property>
     <property name="indent" >
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="secondSeparator" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="actionsLabel" >
     <property name="text" >
      <string>ActionsLabel</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
